synapse/cores/common.py

```diff
@@ -907,12 +904,10 @@ def delTufoTag(self, tufo, tag):
         props = s_tags.getTufoSubs(tufo,tag)
```

Comment:
I was able to rewrite genTufoRows to remove the need for choptag, but genTufoSubs adds tag props that it never had the short name for:
```props.extend([ p for p in tufo[1].keys() if p.startswith(pref) ])```